### DISCUSSION TITLE
tine podcast: 11 – Single Sign On

### DISCUSSION THREAD

**Discussion by pschuele** [tine20/tine20#7474] [General]
In der neuen Episode geht es um ein Akronym, also ein ganz bestimmtes: SSO. Denn tine beherrscht auch beim Single Sign On diverse Rollen. Was sich hinter SSO verbirgt, was das mit tine zu tun hat und warum das alles sehr praktisch ist, erklärt Cornelius Weiss. Ganz nebenbei erzählt er dabei noch von ein paar Neuigkeiten, die für einige in der Community sehr interessant sein könnten. Achso, in dem Zusammenhang haben wir Euch einen Link versprochen, hier ist er: https://tine-docu.s3web.rz1.metaways.net/ (aber erst die Episode hören!).

https://www.tine-groupware.de/podcast/11-single-sign-on/

**Comment by lab-at-nohl** [comment]
Wo kann ich den aktuellen Stand des neuen Dokumentationssystems sehen? 

P.S Ich höre euren Podcast regelmäßig und liebe die Infos zwischen den Zeilen...

**Reply by pschuele** [reply]
> Wo kann ich den aktuellen Stand des neuen Dokumentationssystems sehen?

im github ist das hier zu finden: https://github.com/tine-groupware/tine/tree/main/docs
